Islandora-CLAW/islandora_demo
# Removing rdf mapping entry for field_linked_agent, and it's now confi…

…gurable

**GitHub Issue**: Resolves https://github.com/Islandora-CLAW/CLAW/issues/889

# What does this Pull Request do?

Removes the default rdf mapping entry for `field_linked_agent` because it's now configurable, see: https://github.com/Islandora-CLAW/controlled_access_terms/pull/16

Typed relation fields now add their predicates in an alter, which results in duplicate entries if you have defined an rdf mapping.  

# How should this be tested?

A description of what steps someone could take to:
* Make a Respository Item
* Make a Person
* Edit the Repository Item and add the Person as a linked agent and set the marc relator to use
* Check the jsonld.  Here's what I did (edited for brevity):
```json
{
  "@graph": [
    {
      "@id": "http://localhost:8000/node/1?_format=jsonld",
      "@type": [
        "http://pcdm.org/models#Object"
      ],
...
      "http://id.loc.gov/vocabulary/relators/ctb": [
        {
          "@id": "http://localhost:8000/taxonomy/term/25?_format=jsonld"
        }
      ],
      "http://id.loc.gov/vocabulary/relators/auc": [
        {
          "@id": "http://localhost:8000/taxonomy/term/25?_format=jsonld"
        }
...
  ]
}
```
* Notice that `http://localhost:8000/taxonomy/term/25?_jsonld` is listed twice.  Once as `marc:cbt` and once as `marc:auc`
* Pull in this PR
* `drush fim -y islandora_demo`
* `drush cr`
* Go visit your node again, and the duplicate `marc:ctb` should be gone.  Only what you configure should wind up in the jsonld.

# Interested parties
@Islandora-CLAW/committers
